pallene/coder.lua

```diff
@@ -308,7 +310,7 @@ function Coder:pallene_entry_point_declaration(f_id)
     return (util.render([[
         static ${ret_type} ${name}(
             ${args}
-        )]], {
+        )]], { -- don't leave whitespace between ")" and upcoming ";"
```

Comment:
I didn't understand this comment.

Reply:
If we put the `]]` on the next line, our function declarations would end up looking like the following, with the `)` and `;` on separate lines

```
static void function_01(
    lua_State *L,
    Udata *G,
    StackValue *base
)
;
```